services/QuillLMS/engines/evidence/app/controllers/evidence/research/gen_ai/trials_controller.rb

```diff
@@ -64,15 +64,37 @@ def create
           end
         end
 
+        def update
+          if trial.update(trial_params)
```

Comment:
Since many of the attributes on trial are read-only, I like to use separate params methods to make transparent the difference.  For example,  the LLMs [controller](https://github.com/empirical-org/Empirical-Core/blob/develop/services/QuillLMS/engines/evidence/app/controllers/evidence/research/gen_ai/llms_controller.rb#L27) has `llm_create_params` and `llm_update_params`.  While protection does exist on the model via `attr_readonly` I think the different methods highilight that the params should be different objects.